code/modules/chemistry/tools/food_and_drink.dm

```diff
@@ -1754,6 +1754,14 @@ ADMIN_INTERACT_PROCS(/obj/item/reagent_containers/food/drinks/drinkingglass, pro
 					 "mojito", "cremedementhe", "grasshopper", "freeze", "limeade", "juice_peach",
 					 "juice_banana")
 
+/obj/item/reagent_containers/food/drinks/drinkingglass/icewater
+	New()
+		..()
+		SPAWN(0)
+			if (src.reagents)
+				src.reagents.add_reagent("ice", 15, null, T0C)
+				src.reagents.add_reagent("water", 35, null, T0C)
```

Comment:
Super minor nitpick, but the water being at 0 celsius is weird.

Reply:
Ice got changed recently, and ice and water can now coexist with a little cajoling (have to both be near freezing before mixing)
Testing showed that it remained as both ice and water after spawning in. Liquid in the screenshots above would be a lot whiter if it was all ice